<ui version="4.0" >
 <class>MovingOptionsPage</class>
 <widget class="QWidget" name="MovingOptionsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>563</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="8" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>378</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" >
    <widget class="QLineEdit" name="move_additional_files_pattern" />
   </item>
   <item row="5" column="0" >
    <widget class="QCheckBox" name="move_additional_files" >
     <property name="text" >
      <string>Move additional files:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>2</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="move_files_to" />
     </item>
     <item>
      <widget class="QPushButton" name="move_files_to_browse" >
       <property name="text" >
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="delete_empty_dirs" >
     <property name="text" >
      <string>Delete empty directoriesyyyyy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="copy_files" >
     <property name="text" >
      <string>Copy Files (leave source files untouched)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="move_files" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Move files to this directory when saving:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
